csharp/extractor/Semmle.Extraction.CSharp/Entities/Constructor.cs

```diff
@@ -10,8 +12,16 @@ namespace Semmle.Extraction.CSharp.Entities
 {
     internal class Constructor : Method
     {
+        private readonly Lazy<List<SyntaxNode>> DeclaringReferenceSyntax;
```

Comment:
What do we gain with `Lazy<>`? I think we're using these nodes in the `ReportingLocation`. Doesn't this mean that we need the `DeclaringReferenceSyntax` for all constructors?

Minor: fields should start with lower case letters.

Reply:
Good point, I will just initialize it directly in the constructor.